### PR TITLE
Adds account owned tokens getting started

### PR DESCRIPTION
**Pull request by bnelz**
### Summary

Adds Account Owned Tokens beta Getting Started documentation.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+
+[]
+
+---
+
+## What are Account Owned Tokens?
+
+Account Owned Tokens are a first step that Cloudflare is taking in order to represent service principals in our service, and they are being released currently as an Open Beta.
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
Account Owned Tokens are Cloudflare's first step to represent service principals in our service. Account Owned Tokens are currently in Open Beta.
```

What is "in our service" referring to?

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+Engineers across the company are working to ensure that all features eventually become compatible with Account Owned Tokens, and we have provided a short compatability list below.
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
Cloudflare is working to enable all features to be compatible with Account Owned tokens, and a short compatibility list is provided below.
```

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+If you are working with a service that is not currently supported by account owned tokens please continue to use the existing user tokens.
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
If you are working with a product or feature that is not currently supported by Account Owned tokens, continue to use the existing user tokens.
```

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+User tokens will continue to work, and we do not have plans to deprecate them.
+
+Account Owned Tokens will be visible for Super Administrators of accounts on the Cloudflare dashboard via **Manage Account** > **API Tokens**.
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
Account Owned Tokens are visible for Super Administrators of accounts on the Cloudflare dashboard via **Manage Account** > **API Tokens**.
```

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+The existing token creation UI is available, and can also be accessed from the API at /accounts/:accountID/tokens
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
The existing token creation UI is available and can also be accessed from the API at /accounts/:accountID/tokens.
```

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+Try using Account Owned Tokens specifically in these scenarios:
+
+1. You require business continuity when managing tokens as a team of super administrators
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
1. If you require business continuity when managing tokens as a team of super administrators
```

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+2. You need to restrict API access on your account, and want to centralize visibility and management of these tokens
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
2. If you need to restrict API access on your account and want to centralize visibility and management of these tokens
```

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+## Account Owned Tokens Compatibility Matrix
+
+Account Owned Tokens are a new credential type that is in Open Beta. This is the list of currently supported products and their compatability status.
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx:24]: Just noting that lists like this tend to get out of date quickly - may be a good idea to consider (after birthday week!) figuring out how to dynamically update

File: src/content/partials/fundamentals/account-owned-tokens.mdx (hunk 1 above)
Comment (bnelz) [reply src/content/partials/fundamentals/account-owned-tokens.mdx]: The Cloudflare system.

File: src/content/partials/fundamentals/account-owned-tokens.mdx [hunk 9]
```diff
@@ -6,18 +6,18 @@
 
 ## What are Account Owned Tokens?
 
-Account Owned Tokens are a first step that Cloudflare is taking in order to represent service principals in our service, and they are being released currently as an Open Beta.
-Engineers across the company are working to ensure that all features eventually become compatible with Account Owned Tokens, and we have provided a short compatability list below.
-If you are working with a service that is not currently supported by account owned tokens please continue to use the existing user tokens.
+Account Owned Tokens are Cloudflare's first step to represent service principals in our service. Account Owned Tokens are currently in Open Beta.
+Cloudflare is working to enable all features to be compatible with Account Owned tokens, and a short compatibility list is provided below.
+If you are working with a product or feature that is not currently supported by Account Owned tokens, continue to use the existing user tokens.
```
Comment (crwaters16) [inline src/content/partials/fundamentals/account-owned-tokens.mdx]:
```suggestion
If you are working with a product or feature that is not currently supported by Account Owned Tokens, continue to use the existing user tokens.
```

**Review by crwaters16 — APPROVED** [review]

**Pull request merged**